XenkoShaderExplorer/MainViewModel.cs

```diff
@@ -64,7 +65,16 @@ public MainViewModel()
         {
             try
             {
-                RootShaders = BuildShaderTree().ToList();
+				_path = Directory
```

Comment:
It automatically figures out the correct path. (It takes the first path that starts with the word "Xenko")